tests/testthat/test-gaussian.R

```diff
@@ -54,9 +66,19 @@ test_that("we can approximate the closed form ridge regression solution", {
                        intercept = FALSE,
                        thresh = 0.00001,
                        maxit = 1000)
+  
+  batch_fit <- sgdnet(x, y,
+                       alpha = 0,
+                       lambda = sd_y*lambda/n,
+                       intercept = FALSE,
+                       thresh = 0.00001,
+                       maxit = 1000,
+                       batchsize = 10)
 
   expect_equivalent(beta_theoretical, coef(sgdnet_fit)[-1],
                     tolerance = 1e-3)
+  expect_equivalent(beta_theoretical, coef(batch_fit)[-1],
+                    tolerance = 1e-3)
```

Comment:
Can we add tests for the full batch case where we know it should converge to more or less exactly the ols fit?

Reply:
Hi, Michael, the full batch case to compare with ols fit is now included.